packages/ove-service-auth/README.md

```diff
@@ -0,0 +1,191 @@
+# OVE Base Proxy
+
+This is a generic base proxy that supports various configurations, provided either via environment variables or 
+configuration files.
+
+## Configurations
+
+### Request Proxy
+
+The request proxy supports two configurations: a global or service-based proxy. The service proxy supports basic
+load balancing.
+
+The proxy mechanism can be enabled by setting the **SERVICE_PROXY_ENABLED** environment variable, the default is **True**.
+
+In proxy mode the service maps all requests based on a path matching strategy:
+
+```python
+r'/(?P<service>.+)/(?P<path>.*)'
```

Comment:
The regex syntax may be unfamiliar to some readers. Consider instead writing `/<service>/<path>`, and stating that `<path>` may include additional slashes.